node/cli/src/command.rs

```diff
@@ -73,15 +73,16 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 		"frequency" => return Ok(Box::new(chain_spec::frequency::load_frequency_spec())),
 		#[cfg(feature = "frequency-no-relay")]
 		"dev" | "frequency-no-relay" =>
-			return Ok(Box::new(chain_spec::frequency_rococo::development_config())),
+			return Ok(Box::new(chain_spec::frequency_dev::development_config())),
```

Comment:
Moved the development config out of rococo

Reply:
💯